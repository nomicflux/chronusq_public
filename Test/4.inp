$geom
2
c 0 0.0  0.1 0.0
c 0 1.0  0.0 0.15

$charge 0
$spin 1
$basis sto3g.gbs
$method hf
$print 3
$guess gaufchk 4.fchk


