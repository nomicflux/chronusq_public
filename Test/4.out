Job started: Wed Apr  8 17:45:33 2015


Cartesian coordinates (bohr):
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                                      X              Y              Z
--------------------------------------------------------------------------------
       1       C       6     0.00000000     0.18897261     0.00000000
       2       C       6     1.88972613     0.00000000     0.28345892
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

Molecular Information:
       nAtoms =       2                 Charge =       0
   nElectrons =      12

Basis Function Information:
       nBasis =      10             nPrimitive =      30
      nSShell =       4                nPShell =       2
      nDShell =       0                nFShell =       0
      nGShell =       0

Cartesian Basis Functions:
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
          Center  lx  ly  lz          exponent       coefficient
--------------------------------------------------------------------------------
    1   S    1     0   0   0         71.616837          0.154329
                                     13.045096          0.535328
                                      3.530512          0.444635

    2   S    1     0   0   0          2.941249         -0.099967
                                      0.683483          0.399513
                                      0.222290          0.700115

    3   P    1     1   0   0          2.941249          0.155916
                                      0.683483          0.607684
                                      0.222290          0.391957
                   0   1   0          2.941249          0.155916
                                      0.683483          0.607684
                                      0.222290          0.391957
                   0   0   1          2.941249          0.155916
                                      0.683483          0.607684
                                      0.222290          0.391957

    4   S    2     0   0   0         71.616837          0.154329
                                     13.045096          0.535328
                                      3.530512          0.444635

    5   S    2     0   0   0          2.941249         -0.099967
                                      0.683483          0.399513
                                      0.222290          0.700115

    6   P    2     1   0   0          2.941249          0.155916
                                      0.683483          0.607684
                                      0.222290          0.391957
                   0   1   0          2.941249          0.155916
                                      0.683483          0.607684
                                      0.222290          0.391957
                   0   0   1          2.941249          0.155916
                                      0.683483          0.607684
                                      0.222290          0.391957
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

Atmoic Orbital Shell Pairs:
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
   Index    Center-1    Shell-1   L-1     *  Center-2    Shell-2   L-2
--------------------------------------------------------------------------------
       1           1          3     1     *         1          3     1
       2           1          3     1     *         2          6     1
       3           1          3     1     *         1          1     0
       4           1          3     1     *         1          2     0
       5           1          3     1     *         2          4     0
       6           1          3     1     *         2          5     0
       7           2          6     1     *         2          6     1
       8           2          6     1     *         1          1     0
       9           2          6     1     *         1          2     0
      10           2          6     1     *         2          4     0
      11           2          6     1     *         2          5     0
      12           1          1     0     *         1          1     0
      13           1          1     0     *         1          2     0
      14           1          1     0     *         2          4     0
      15           1          1     0     *         2          5     0
      16           1          2     0     *         1          2     0
      17           1          2     0     *         2          4     0
      18           1          2     0     *         2          5     0
      19           2          4     0     *         2          4     0
      20           2          4     0     *         2          5     0
      21           2          5     0     *         2          5     0
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

Single Slater Determinent Wave Function Information:
        nOccA =       6                  nVirA =       4
        nOccB =       6                  nVirB =       4
 Multiplicity =       1
reading formatted checkpoint file 4.fchk

Alpha Molecular Orbital Coefficients :
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                   1              2              3              4              5
    1   0.7011827300   0.7032857950  -0.2168307940  -0.0000000000   0.0000000000
    2   0.0142922628   0.0352651355   0.4832005640   0.0000000000  -0.0000000000
    3   0.0046190200   0.0117141474   0.2202869100  -0.1028559780   0.0246058314
    4  -0.0004619020  -0.0011714147  -0.0220286910  -0.2010838750   0.5580826670
    5   0.0006928530   0.0017571221   0.0330430365   0.5516506030   0.2080162360
    6   0.7011827300  -0.7032857950  -0.2168307940   0.0000000000  -0.0000000000
    7   0.0142922628  -0.0352651355   0.4832005640  -0.0000000000   0.0000000000
    8  -0.0046190200   0.0117141474  -0.2202869100  -0.1028559780   0.0246058314
    9   0.0004619020  -0.0011714147   0.0220286910  -0.2010838750   0.5580826670
   10  -0.0006928530   0.0017571221  -0.0330430365   0.5516506030   0.2080162360

                   6              7              8              9             10
    1  -0.1744423740   0.0351297646  -0.0000000000   0.0000000000  -0.1068597250
    2   0.7142318990  -0.3884978970   0.0000000000  -0.0000000000   2.2114096700
    3  -0.3192127760   0.6354417920   0.1036961640  -0.1256722920   1.6315959700
    4   0.0319212776  -0.0635441792   0.9084411430   0.0996287326  -0.1631595970
    5  -0.0478819164   0.0953162687  -0.0856803303   0.9042344350   0.2447393950
    6   0.1744423740   0.0351297646  -0.0000000000   0.0000000000   0.1068597250
    7  -0.7142318990  -0.3884978970   0.0000000000  -0.0000000000  -2.2114096700
    8  -0.3192127760  -0.6354417920  -0.1036961640   0.1256722920   1.6315959700
    9   0.0319212776   0.0635441792  -0.9084411430  -0.0996287326  -0.1631595970
   10  -0.0478819164  -0.0953162687   0.0856803303  -0.9042344350   0.2447393950
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

Alpha Density :
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                   1              2              3              4              5
    1   2.1274277307
    2  -0.3890842266   1.4901157783
    3   0.0387928387  -0.2421401416   0.3235329496
    4  -0.0038792839   0.0242140142   0.0387131981   0.7067936095
    5   0.0058189258  -0.0363210212  -0.0580697970   0.0058069795   0.7019544599
    6   0.0272635256   0.0100791302  -0.2168976729   0.0216897673  -0.0325346509
    7   0.0100791302  -0.5553675630   0.6681752432  -0.0668175243   0.1002262865
    8   0.2168976729  -0.6681752432   0.1293423174   0.0581322614  -0.0871983918
    9  -0.0216897673   0.0668175243   0.0581322614   0.7048517032   0.0087198390
   10   0.0325346509  -0.1002262865  -0.0871983918   0.0087198390   0.6975851706

                   6              7              8              9             10
    6   2.1274277307
    7  -0.3890842266   1.4901157783
    8  -0.0387928387   0.2421401416   0.3235329496
    9   0.0038792839  -0.0242140142   0.0387131981   0.7067936095
   10  -0.0058189258   0.0363210212  -0.0580697970   0.0058069795   0.7019544599
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

CPU time for building the Coulomb matrix:0 seconds.

Alpha Coulomb Integral :
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                   1              2              3              4              5
    1   6.6947109557
    2   1.3430852976   3.4739340773
    3   0.0479478509   0.4239884612   3.6106434048
    4  -0.0047947851  -0.0423988461  -0.0213043974   3.3997298702
    5   0.0071921776   0.0635982692   0.0319565961  -0.0031956596   3.4023929199
    6   0.0015084117   0.4968195983   0.8310721240  -0.0831072124   0.1246608186
    7   0.4968195983   2.1177283905   1.9323303840  -0.1932330384   0.2898495576
    8  -0.8310721240  -1.9323303840  -1.1187533042   0.2474975017  -0.3712462525
    9   0.0831072124   0.1932330384   0.2474975017   1.3314719623   0.0371246252
   10  -0.1246608186  -0.2898495576  -0.3712462525   0.0371246252   1.3005347746

                   6              7              8              9             10
    6   6.6947109557
    7   1.3430852976   3.4739340773
    8  -0.0479478509  -0.4239884612   3.6106434048
    9   0.0047947851   0.0423988461  -0.0213043974   3.3997298702
   10  -0.0071921776  -0.0635982692   0.0319565961  -0.0031956596   3.4023929199
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

CPU time for building the Exchange matrix:0 seconds.

Alpha Exchange Integral :
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                   1              2              3              4              5
    1   1.8031237092
    2   0.3089425193   0.4533709528
    3   0.0044863383   0.0996596631   0.3066264039
    4  -0.0004486338  -0.0099659663  -0.0000142423   0.3064854052
    5   0.0006729507   0.0149489495   0.0000213634  -0.0000021364   0.3064871856
    6   0.0004707991   0.1175931459   0.1902586438  -0.0190258644   0.0285387966
    7   0.1175931459   0.3028961683   0.2762196329  -0.0276219633   0.0414329449
    8  -0.1902586438  -0.2762196329  -0.0623137222   0.0277507551  -0.0416261327
    9   0.0190258644   0.0276219633   0.0277507551   0.2124187534   0.0041626132
   10  -0.0285387966  -0.0414329449  -0.0416261327   0.0041626132   0.2089499090

                   6              7              8              9             10
    6   1.8031237092
    7   0.3089425193   0.4533709528
    8  -0.0044863383  -0.0996596631   0.3066264039
    9   0.0004486338   0.0099659663  -0.0000142423   0.3064854052
   10  -0.0006729507  -0.0149489495   0.0000213634  -0.0000021364   0.3064871856
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

Overlap :
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                   1              2              3              4              5
    1   1.0000000000
    2   0.2483623971   1.0000000000
    3   0.0000000000   0.0000000000   1.0000000000
    4   0.0000000000   0.0000000000   0.0000000000   1.0000000000
    5   0.0000000000   0.0000000000   0.0000000000   0.0000000000   1.0000000000
    6   0.0003073414   0.0897680060   0.1491746818  -0.0149174682   0.0223762023
    7   0.0897680060   0.5802961601   0.4910050451  -0.0491005045   0.0736507568
    8  -0.1491746818  -0.4910050451  -0.2102017561   0.0617334648  -0.0926001972
    9   0.0149174682   0.0491005045   0.0617334648   0.4009595457   0.0092600197
   10  -0.0223762023  -0.0736507568  -0.0926001972   0.0092600197   0.3932428626

                   6              7              8              9             10
    6   1.0000000000
    7   0.2483623971   1.0000000000
    8   0.0000000000   0.0000000000   1.0000000000
    9   0.0000000000   0.0000000000   0.0000000000   1.0000000000
   10   0.0000000000   0.0000000000   0.0000000000   0.0000000000   1.0000000000
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

Kinetic :
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                   1              2              3              4              5
    1  15.8911218138
    2  -0.0858899941   0.4722499926
    3   0.0000000000   0.0000000000   1.4777280906
    4   0.0000000000   0.0000000000   0.0000000000   1.4777280906
    5   0.0000000000   0.0000000000   0.0000000000   0.0000000000   1.4777280906
    6  -0.0058309901   0.0066401241   0.0678613569  -0.0067861357   0.0101792035
    7   0.0066401241   0.1539696457   0.3088735697  -0.0308873570   0.0463310355
    8  -0.0678613569  -0.3088735697  -0.4866321887   0.0747544565  -0.1121316848
    9   0.0067861357   0.0308873570   0.0747544565   0.2534369309   0.0112131685
   10  -0.0101792035  -0.0463310355  -0.1121316848   0.0112131685   0.2440926239

                   6              7              8              9             10
    6  15.8911218138
    7  -0.0858899941   0.4722499926
    8   0.0000000000   0.0000000000   1.4777280906
    9   0.0000000000   0.0000000000   0.0000000000   1.4777280906
   10   0.0000000000   0.0000000000   0.0000000000   0.0000000000   1.4777280906
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

Potential :
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                   1              2              3              4              5
    1  36.7774341880
    2   4.9682850258   8.1925879330
    3   0.1150742147   1.1258139635   8.7014414183
    4  -0.0115074215  -0.1125813963  -0.0838690389   7.8711379333
    5   0.0172611322   0.1688720945   0.1258035583  -0.0125803558   7.8816215632
    6   0.0039199444   1.8631966233   3.1446840643  -0.3144684064   0.4717026096
    7   1.8631966233   5.0127112324   4.6515345714  -0.4651534571   0.6977301857
    8  -3.1446840643  -4.6515345714  -2.7616885982   0.5772937852  -0.8659406778
    9   0.3144684064   0.4651534571   0.5772937852   2.9535198754   0.0865940678
   10  -0.4717026096  -0.6977301857  -0.8659406778   0.0865940678   2.8813581523

                   6              7              8              9             10
    6  36.7774341880
    7   4.9682850258   8.1925879330
    8  -0.1150742147  -1.1258139635   8.7014414183
    9   0.0115074215   0.1125813963  -0.0838690389   7.8711379333
   10  -0.0172611322  -0.1688720945   0.1258035583  -0.0125803558   7.8816215632
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

One Electron Integral :
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                   1              2              3              4              5
    1 -20.8863123742
    2  -5.0541750199  -7.7203379404
    3  -0.1150742147  -1.1258139635  -7.2237133277
    4   0.0115074215   0.1125813963   0.0838690389  -6.3934098427
    5  -0.0172611322  -0.1688720945  -0.1258035583   0.0125803558  -6.4038934725
    6  -0.0097509345  -1.8565564992  -3.0768227074   0.3076822707  -0.4615234061
    7  -1.8565564992  -4.8587415867  -4.3426610017   0.4342661002  -0.6513991503
    8   3.0768227074   4.3426610017   2.2750564095  -0.5025393287   0.7538089930
    9  -0.3076822707  -0.4342661002  -0.5025393287  -2.7000829445  -0.0753808993
   10   0.4615234061   0.6513991503   0.7538089930  -0.0753808993  -2.6372655284

                   6              7              8              9             10
    6 -20.8863123742
    7  -5.0541750199  -7.7203379404
    8   0.1150742147   1.1258139635  -7.2237133277
    9  -0.0115074215  -0.1125813963   0.0838690389  -6.3934098427
   10   0.0172611322   0.1688720945  -0.1258035583   0.0125803558  -6.4038934725
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

CPU time for one-electron integral:0 seconds.

Alpha Fock :
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                   1              2              3              4              5
    1 -15.9947251278
    2  -4.0200322416  -4.6997748159
    3  -0.0716127021  -0.8014851653  -3.9196963268
    4   0.0071612702   0.0801485165   0.0625788837  -3.3001653777
    5  -0.0107419053  -0.1202227748  -0.0938683256   0.0093868326  -3.3079877382
    6  -0.0087133219  -1.4773300468  -2.4360092271   0.2436009227  -0.3654013841
    7  -1.4773300468  -3.0439093645  -2.6865502506   0.2686550251  -0.4029825376
    8   2.4360092271   2.6865502506   1.2186168274  -0.2827925821   0.4241888732
    9  -0.2436009227  -0.2686550251  -0.2827925821  -1.5810297356  -0.0424188873
   10   0.3654013841   0.4029825376   0.4241888732  -0.0424188873  -1.5456806628

                   6              7              8              9             10
    6 -15.9947251278
    7  -4.0200322416  -4.6997748159
    8   0.0716127021   0.8014851653  -3.9196963268
    9  -0.0071612702  -0.0801485165   0.0625788837  -3.3001653777
   10   0.0107419053   0.1202227748  -0.0938683256   0.0093868326  -3.3079877382
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

Energy Information:
            E(one electron) = -136.7489720361 a.u. 
            E(two electron) =   43.7242599788 a.u. 
       E(nuclear repulsion) =   18.7481578510 a.u. 
                   E(total) =  -74.2765542062 a.u. 

Job finished: Wed Apr  8 17:45:33 2015

